src/__tests__/RenderParentDashboard.test.js

```diff
@@ -33,26 +34,31 @@ const Component = props => {
 };
 
 describe('<ParentDashboard /> test suite', () => {
-  test('Welcome Back', () => {
+  test('Welcome back Nav', () => {
     render(<Component parent={{ children: [] }} />);
-    expect(screen.getByText(/Welcome Back/i)).toBeInTheDocument();
-  });
 
-  test('Dashboard Nav', () => {
-    render(<Component parent={{ children: [] }} />);
-    expect(screen.getByText(/Dashboard/i)).toBeInTheDocument();
+    expect(screen.getByText(/Welcome back/i)).toBeInTheDocument();
   });
 
-  test('Help Nav', () => {
-    render(<Component parent={{ children: [] }} />);
-    expect(screen.getByText(/Help/i)).toBeInTheDocument();
-  });
-  test('Parent Settings Nav', () => {
+  test('Change user Nav', () => {
     render(<Component parent={{ children: [] }} />);
-    expect(screen.getByText(/Parent Settings/i)).toBeInTheDocument();
+
+    fireEvent(
+      screen.getByTestId('parent-avatar'),
+      new MouseEvent('click', { bubbles: true, cancelable: true })
+    );
+
```

Comment:
Correct me if I am wrong but it looks like these tests only check if the UI is rendered correctly to the page, is there a reason you didn't also test the functionality?

Reply:
That's a really good point to raise! Testing functionality should be included--I agree. As I was rewriting the test suite for this new iteration, I was following the testing patterns used on the defunct NavSider component (which, for whatever reason, had no testing of functionality.)